Corona-Warn-App/src/main/res/layout/include_submission_positive_other_warning.xml

```diff
@@ -51,18 +56,39 @@
                 app:layout_constraintStart_toStartOf="@id/guideline_start"
                 app:layout_constraintTop_toBottomOf="@+id/submission_positive_other_warning_headline" />
 
+            <TextView
+                android:id="@+id/submission_country_header_description"
+                style="@style/headline6"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="24dp"
+                android:text="@{@string/submission_interoperability_list_title}"
+                app:layout_constraintEnd_toEndOf="@id/guideline_end"
+                app:layout_constraintStart_toStartOf="@id/guideline_start"
+                app:layout_constraintTop_toBottomOf="@+id/submission_positive_other_warning_text" />
+
+            <de.rki.coronawarnapp.ui.view.CountryList
+                android:id="@+id/countryList"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/spacing_normal"
+                app:layout_constraintEnd_toEndOf="@+id/submission_country_header_description"
+                app:layout_constraintStart_toStartOf="@+id/submission_country_header_description"
+                app:layout_constraintTop_toBottomOf="@+id/submission_country_header_description"
+                app:list="@{countryData}" />
+
             <include
                 android:id="@+id/submission_positive_location_card_16_years"
                 layout="@layout/include_16_years"
                 android:layout_width="@dimen/match_constraint"
                 android:layout_height="wrap_content"
-                android:layout_marginTop="@dimen/spacing_large"
+                android:layout_marginTop="48dp"
```

Comment:
Please use @dimen/spacing_large as you did in your previous commit it instead of hard coded values

Reply:
Weird, haven't touched this but I'll change it back.